### ISSUE TITLE
Cannot serialize body while using default exception handler

### ISSUE THREAD

**Issue by pablobcb** [metosin/compojure-api#313]
I'm trying to log the request body of every request that raises an unhandled exception

```clojure
(def app
  (api
   {:exceptions
    {:handlers
     {:compojure.api.exception/default
      (fn [^Exception e data request]
        (prn (ring.util.request/body-string request)))}}}

   (context "/" []
     (POST "/error" [] (/ 1 0))))
```

right now im getting `""` instead of the request body. Am I doing something wrong?

**Comment by ikitommi** [comment]
Hi. I think there are two issues:

1) the exception handler should return a ring-response, return something like `{:status 500, :body "..."}`

2) The body is a `InputStream` in the ring-model, it can only be consumed once. When you have hit the exception handler, if the payload was something that compojure-api automatically decodes, the `InputStream` is empty and the body is parsed and found from `:body-params`. 

This should work:

```clj
(def app
  (api
   {:exceptions
    {:handlers
     {:compojure.api.exception/default
      (fn [^Exception e data request]
        (internal-server-error {:body (:body-params request)}}}
   (context "/" []
     (POST "/error" [] (/ 1 0))))
```

**Comment by pablobcb** [comment]
actually it doesnt :(

using exactly the snippet you posted and using this request

```
POST /error HTTP/1.1
Host: localhost:3000
Content-Type: application/json; charset=utf-8
Authorization: Basic bla-ble-bli-blo-blu
Cache-Control: no-cache
Postman-Token: d80d7f35-4e3a-6971-093f-35e4f60b4d05

{
  "a": "1111111",
  "b": "222222"
}
```

and got 
```
{
    "body": null
}
```
 as response

**Comment by pablobcb** [comment]
fixed on "2.0.0-alpha3"
